### PR TITLE
CLDR-17658 Virtual scroller for Dashboard

### PR DESCRIPTION
**Pull request by btangmu**
- Use vue-virtual-scroller
- Remove recently added CATS_ONE_PER_PAGE feature
 
CLDR-17658

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

File: tools/cldr-apps/js/package.json [hunk 1]
```diff
@@ -48,6 +48,7 @@
     "marked": "^4.3.0",
     "swagger-client": "^3.26.7",
     "vue": "^3.2.47",
+    "vue-virtual-scroller": "^2.0.0-beta.8",
```
Comment (btangmu) [inline tools/cldr-apps/js/package.json:51]: IMPORTANT: this is a "beta" version

**Comment by btangmu** [comment]
This is deployed on cldr-staging currently

**Comment by srl295** [comment]
performance seems great. A little blinky, perhaps the controls at the top are being more dynamic than they need to be?

**Comment by btangmu** [comment]
> performance seems great. A little blinky, perhaps the controls at the top are being more dynamic than they need to be?

I've noticed for a while now that sometimes the header appears then disappears then reappears -- if that's what you mean by "blinky", it's annoying and predates the virtual scroller; something about the sequence of events while loading, maybe

**Pull request merged**